policyengine_us/variables/gov/states/ar/tax/income/credits/ar_additional_tax_credit_for_qualified_individuals_person.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_us.model_api import *
+
+
+class ar_additional_tax_credit_for_qualified_individuals_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Arkansas additional tax credit for qualified individuals for each individual"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameters):
+        filing_separately = person.tax_unit("ar_files_separately", period)
+        income_joint = person("ar_taxable_income_joint", period)
+        income_indiv = person("ar_taxable_income_indiv", period)
+        # When filing separartely, the credit is calculated based on individual income
+        income = where(
+            filing_separately, income_indiv, person.tax_unit.sum(income_joint)
+        )
+        p = parameters(
+            period
+        ).gov.states.ar.tax.income.credits.additional_tax_credit_for_qualified_individuals
+        filing_status = person.tax_unit("filing_status", period)
+        joint = filing_status == filing_status.possible_values.JOINT
+        filing_jointly = joint & ~filing_separately
+        max_amount = where(
```

Comment:
same for reduction_amount, or i think you could pull multiplier out in front of both instead
```suggestion
        multiplier = where(filing_jointly, p.joint_multiplier, 1)
        max_amount = p.max_amount * multiplier
```